manifest.json

```diff
@@ -11,12 +11,18 @@
     "128": "images/reddit-emoji-128.png"
   },
   "background": {
-    "service_worker": "scripts/background.js"
+    "service_worker": "scripts/background.js",
+    "scripts": ["scripts/background.js"]
+  },
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "{6dcd5876-58a7-481f-ac1c-c1c3ee062a75}"
+    }
   },
   "web_accessible_resources": [
     {
       "resources": ["css/*.css"],
-      "extension_ids": ["kfbkibjjmfhcgdnhjcpjjmceealpmikl"]
+      "matches": ["*://www.reddit.com/*"]
```

Comment:
Here to replace the `extension_ids` field since Firefox and Chrome expect different formats for these id's.